tests-semanticdb/src/test/resources/metac.expect_2.13

```diff
@@ -82,6 +82,79 @@ Diagnostics:
 [42:4..42:8) [warning] a pure expression does nothing in statement position; multiline expressions might require enclosing parentheses
 [50:4..50:8) [warning] a pure expression does nothing in statement position; multiline expressions might require enclosing parentheses
 
+semanticdb/integration/src/main/scala-2.13/example/SingleAbstractMethod.scala
+-----------------------------------------------------------------------------
+
+Summary:
+Schema => SemanticDB v4
+Uri => semanticdb/integration/src/main/scala-2.13/example/SingleAbstractMethod.scala
+Text => non-empty
+Language => Scala
+Symbols => 14 entries
+Occurrences => 30 entries
+
+Symbols:
+example/SingleAbstractMethod# => class SingleAbstractMethod extends AnyRef { +4 decls }
+  AnyRef => scala/AnyRef#
+example/SingleAbstractMethod#SAM# => trait SAM extends AnyRef { +1 decls }
+  AnyRef => scala/AnyRef#
+example/SingleAbstractMethod#SAM#foo(). => abstract method foo(a: Int): Int
+  a => example/SingleAbstractMethod#SAM#foo().(a)
+  Int => scala/Int#
+example/SingleAbstractMethod#SAM#foo().(a) => param a: Int
+  Int => scala/Int#
+example/SingleAbstractMethod#`<init>`(). => primary ctor <init>()
+example/SingleAbstractMethod#funcSAM(). => method funcSAM(y: Int): Int
+  y => example/SingleAbstractMethod#funcSAM().(y)
+  Int => scala/Int#
+example/SingleAbstractMethod#funcSAM().(y) => param y: Int
+  Int => scala/Int#
+example/SingleAbstractMethod#withSAM(). => method withSAM(a: SAM): Int
+  a => example/SingleAbstractMethod#withSAM().(a)
+  SAM => example/SingleAbstractMethod#SAM#
+  Int => scala/Int#
+example/SingleAbstractMethod#withSAM().(a) => param a: SAM
+  SAM => example/SingleAbstractMethod#SAM#
+local0 => val local $anonfun
```

Comment:
is that a useful symbol?

Reply:
ideally it would point to `SingleAbstractMethod` as a parent such as:

```suggestion
local0 => val local: AnyRef with SingleAbstractMethod {}
  AnyRef => scala/AnyRef#
  SingleAbstractMethod => _empty_/SingleAbstractMethod#
```

Reply:
done... i think.